src/client/components/ProductDetails/ProductDetails.component.js

```diff
@@ -64,7 +69,6 @@ export const ProductDetails = ({
           throw new Error(response.status);
         }
       });
-      window.location.reload(false);
```

Comment:
Please keep this .. so it will render when you remove a product in favorites page

Reply:
Hi, reload of page(window.location.reload.(false)) is not working(see the screenshot). Moreover product details page need not be reloaded when a favourite is added or removed. If the page needs to be reloaded in favourites page then it should be added there. 

![Screenshot 2021-12-14 at 18 48 44](https://user-images.githubusercontent.com/81102420/146053057-936fb901-1cee-46fb-bff0-df6480bd9294.png)

Reply:
I think that reload button has nothing to do this error

Reply:
To show whether a product is a favourite or not, we need user object. But when reloading the page, we are not getting the "auth" object from firebase.

Reply:
Having issues in getting the user uid from firebase when reloading the page. Taking the user uid from local storage now.

Reply:
Added reload to the code.
Reloading the page was giving problem to fetch authenticated user from firebase. Taking the user uid from local storage now.

Reply:
Super